packages/eslint-plugin/src/rules/no-visibility-modifiers.js

```diff
@@ -1,17 +1,23 @@
 // @ts-check
-// @ts-ignore
-const util = require('@typescript-eslint/eslint-plugin/dist/util');
-const { AST_NODE_TYPES } = require('@typescript-eslint/experimental-utils');
+const _internalApiMisc =
```

Comment:
this rule might be probably deleted as this kind of rule comes with ts/eslint plugin. for sake of consistency and compatibilty with new versions I added proper types for this private api.